examples/mongocxx/mongodb.com/documentation_examples.cpp

```diff
@@ -1180,13 +1182,63 @@ void delete_examples(mongocxx::database db) {
     }
 }
 
+void start_versioned_API_examples() {
+    // To start a versioned API, you can set specific options in the mongocxx::client's
+    // server flags:
+    auto server_api = mongocxx::options::server_api(mongocxx::options::server_api::version::k_version_1);
+
+    // There are several helper member functions for converting between strings and API versions:
+    std::string version_string = mongocxx::options::server_api::version_to_string(mongocxx::options::server_api::version::k_version_1);
+
+    // Convert a string to an API version:
+    
+    // Using an invalid version string should throw a mongocxx::logic_error:
+    try
+     {
+         mongocxx::options::server_api::version version = 
```

Comment:
FYI, I've written these types out fully mostly for my own edification-- at the next set of commits (depending on discussion WRT exception types) I'll write them in a more idiomatic (e.g. with auto) fashion.